GitFyle.Core.Api/Models/Foundations/Sources/Source.cs

```diff
@@ -11,5 +12,6 @@ public class Source
         public Guid Id { get; set; }
         public string Name { get; set; }
         public string Url { get; set; }
+        public Repository Repository { get; set; }
```

Comment:
A source (GitHub or ...) can have many repositories, so we need to change this to:

```suggestion
        public IEnumerable<Repository> Repositories { get; set; }
```